pyiron_base/project/generic.py

```diff
@@ -967,17 +967,20 @@ def get_id_from_job_table(job_table, job_path):
         else:
             return None
 
-    def refresh_job_status(self, *jobs):
+    def refresh_job_status(self, *jobs, by_status=["running", "submitted"]):
```

Comment:
I would prefer `by_status=["running"]` for the default behaviour, but I can also live with this.

But actually which code doesn't do the transition from `submitted` to `running` automatically inside the code?

Reply:
It can happend when jobs either crash early inside pyiron or are cancelled by the user via slurm.

Reply:
ok fine by me. I can live with this.